apisix/plugins/error-log-logger.lua

```diff
@@ -36,23 +38,40 @@ local lrucache = core.lrucache.new({
 local metadata_schema = {
     type = "object",
     properties = {
-        host = schema_def.host_def,
```

Comment:
We need to keep the old fields and mark them as deprecated (at least in the doc), like:
https://github.com/apache/apisix/blob/71bc27cc41cf52ba1a41816311412527ae278045/apisix/plugins/authz-keycloak.lua#L34

Reply:
got it.